admin/load.php

```diff
@@ -503,3 +508,129 @@ function perflab_dismiss_wp_pointer_wrapper() {
 	check_ajax_referer( 'dismiss_pointer' );
 }
 add_action( 'wp_ajax_dismiss-wp-pointer', 'perflab_dismiss_wp_pointer_wrapper', 0 );
+
+/**
+ * Callback function to handle admin scripts.
+ *
+ * @since n.e.x.t
+ */
+function perflab_enqueue_modules_page_scripts() {
+	wp_enqueue_script( 'updates' );
+
+	wp_localize_script(
+		'updates',
+		'_wpUpdatesItemCounts',
+		array(
+			'settings' => array(
+				'totals' => wp_get_update_data(),
+			),
+		)
+	);
+
+	wp_enqueue_script( 'thickbox' );
+	wp_enqueue_style( 'thickbox' );
+}
+
+/**
+ * Callback function hooked to admin_action_perflab_activate_plugin to handle plugin activation.
+ *
+ * @since n.e.x.t
+ */
+function perflab_activate_plugin() {
+	// Do not proceed if plugin query arg is not present.
+	if ( empty( $_GET['plugin'] ) ) {
+		return;
+	}
+
+	// The plugin being activated.
+	$plugin = sanitize_text_field( $_GET['plugin'] );
+
+	check_admin_referer( "perflab_activate_plugin_{$plugin}" );
+
+	if ( ! current_user_can( 'activate_plugin', $plugin ) ) {
+		wp_die( __( 'Sorry, you are not allowed to activate this plugin.', 'default' ) ); // phpcs:ignore
```

Comment:
Updated in https://github.com/WordPress/performance/pull/864/commits/6eab09cbe760f27033a58323b03eb2648c650108 and https://github.com/WordPress/performance/pull/864/commits/07d87c327ded5fc5d25108b346cfe540350f2c1e.

For some of the ignore statements (related to escaped output), it actually made sense to address the violation (even though those are very much nit-picking since the same code comes from WordPress core where these messages aren't being escaped).